src/com/rjmetrics/sweet_liberty/core.clj

```diff
@@ -69,14 +70,16 @@
   (assoc-in sweet-lib-config [:liberator-config :handle-exception]
             (fn [ctx]
               (let [exception-info (:exception ctx)]
-                (if (-> exception-info ex-data :is-sweet-lib-exception?)
-                  (do
-                    (log-exception exception-info)
-                    (ring-response {:body (get-error-json exception-info
-                                                          (get-in sweet-lib-config [:options :return-exceptions?]))
-                                    :status 400
-                                    :headers {"Content-Type" "application/json; charset=UTF-8"}}))
-                  (handle-exception-fn ctx))))))
+                (if-not (-> exception-info ex-data :is-sweet-lib-exception?)
```

Comment:
Umm, I'm not sure. What do you think? I think that it probably doesn't make too much of a difference except in the super rare edge case where someone is using sweet-lib and doing something similar to what we're doing and chooses to add a `:ring-response` field to their exception and then we handle it and they have no what happened. If we also check for the `:is-sweet-lib-exception?` then we ensure that someone using this feature intends to.